enterprise/utils.py

```diff
@@ -822,12 +822,12 @@ def is_course_run_upgradeable(course_run):
     return False
 
 
-def is_course_run_published(course_run):
+def is_course_run_unpublished(course_run):
     """
-    Return True if the course run's status value is "published".
+    Return True if the course run's status value is "unpublished".
     """
     if course_run:
-        if course_run.get('status') == 'published':
+        if course_run.get('status') == 'unpublished':
```

Comment:
What are the other statuses?

Reply:
@saleem-latif 
the main purpose of this method was to filter out all unpublished course_runs from the enrollable courses.
it was doing it successfully for all the course_runs where the status key was present, but the course_runs where the status key was missing(as it's not mandatory) this check was returning incorrect value. (None == published)
So just modified this method in a way to handle scenarios where the `status` key is not present to consider that course available for enrollment.
@saleem-latif is it helping to answer your question?

Reply:
this was the PR when we wrote this method.
https://github.com/edx/edx-enterprise/pull/795

Reply:
So for missing status, the course will be considered published, right?